modules/distr-tracing-otel-migrating-from-jaeger-with-sidecars.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-otel-migrating.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-migrating-from-jaeger-with-sidecars_{context}"]
+= Migrating from the {JaegerShortName} to the {OTELShortName} with sidecars
+
+The {OTELShortName} Operator supports sidecar injection into Deployment workloads, so you can migrate from a {JaegerShortName} sidecar to a {OTELShortName} sidecar.
+
+.Prerequisites
+
+* The {JaegerName} is used on the cluster.
+* The {OTELName} is installed.
+
+.Procedure
+
+. Configure the OpenTelemetry Collector as a sidecar.
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: <otel-collector-namespace>
+spec:
+  mode: sidecar
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+    processors:
+      batch:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 50
+        spike_limit_percentage: 30
+      resourcedetection:
+        detectors: [openshift]
+        timeout: 2s
+    exporters:
+      otlp:
+        endpoint: "tempo-<example>-gateway:8090" <1>
+        tls:
+          insecure: true
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger]
+          processors: [memory_limiter, resourcedetection, batch]
+          exporters: [otlp]
+----
+<1> This endpoint points to the Gateway of a TempoStack instance deployed by using the `<example>` {TempoOperator}.
+
+. Create a service account for running your application.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-sidecar
+----
+
+. Create a cluster role for the permissions needed by some processors.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector-sidecar
+rules:
+  <1>
+- apiGroups: ["config.openshift.io"]
+  resources: ["infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+----
+<1> The `resourcedetectionprocessor` requires permissions for infrastructures and infrastructures/status.
+
+. Create a `ClusterRoleBinding` to set the permissions for the service account.
```

Comment:
Per [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references):

>Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability.

Reply:
Adding this to my list. Same reply as in https://github.com/openshift/openshift-docs/pull/62410#discussion_r1321079006.